# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/stream/tentative/abort.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/stream/tentative/abort.any.js.ini
--- a/testing/web-platform/meta/websockets/stream/tentative/abort.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/stream/tentative/abort.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -48,12 +48,9 @@
 
 [abort.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
+    if (os == "android") and not debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and debug and webrender: ["OK", "ERROR", "TIMEOUT"]
   [abort before constructing should prevent connection]
     expected: FAIL
 
@@ -66,12 +63,8 @@
 
 [abort.any.sharedworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR"]
-    if os == "linux": TIMEOUT
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
+    if os == "android": ["OK", "ERROR", "TIMEOUT"]
   [abort before constructing should prevent connection]
     expected: FAIL
 
@@ -84,12 +77,11 @@
 
 [abort.any.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and not webrender and not debug: ["ERROR", "TIMEOUT"]
-    if (os == "android") and not webrender and debug: ["TIMEOUT", "ERROR"]
-    if (os == "android") and webrender and debug: ["TIMEOUT", "CRASH"]
-    if (os == "android") and webrender and not debug: ["TIMEOUT", "ERROR", "CRASH"]
-    if os == "linux": TIMEOUT
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
+    if (os == "android") and debug and webrender: ["OK", "TIMEOUT", "CRASH"]
+    if (os == "android") and debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and not debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and not debug and webrender: ["OK", "ERROR", "TIMEOUT", "CRASH"]
   [abort before constructing should prevent connection]
     expected: FAIL
 
@@ -135,12 +127,9 @@
 
 [abort.any.serviceworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and debug and webrender: ["ERROR", "TIMEOUT"]
-    if (os == "android") and not debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "android") and not debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and debug and webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [abort before constructing should prevent connection]
     expected: FAIL
 